package.json

```diff
@@ -2,9 +2,15 @@
   "name": "courses",
   "version": "1.0.0",
   "private": true,
+  "proxy": "http://localhost:8090",
   "dependencies": {
+    "@fullcalendar/daygrid": "^5.1.0",
+    "@fullcalendar/list": "^5.1.0",
+    "@fullcalendar/react": "^5.1.0",
+    "@fullcalendar/timegrid": "^5.1.0",
     "@material-ui/core": "^4.0.0",
     "@material-ui/icons": "^4.0.0",
+    "@react-pdf/renderer": "^2.0.0-beta.14",
```

Comment:
Are we using the react-pdf renderer?

Reply:
nope. will remove that

Reply:
this was removed along with i18 deps, since those aren't being used anymore either